### ISSUE TITLE
Large files being tracked in git

### ISSUE THREAD

**Issue by bhillmann** [GabeAl/NINJA-OPS#12]
Approximately 700 mb are being tracked on old database files within the git repo. When trying to purge the files on a fork and switch over the database folder to git lfs, I run into this issue:

Git LFS: (0 of 8 files) 0 B / 237.83 MB
Git LFS objects can not be uploaded to this public fork.
Docs: https://github.com/contact
Git LFS objects can not be uploaded to this public fork.
Docs: https://github.com/contact

Since LFS is a part of github and not git, it might be in our best interest to use something else to download the database files.

**Comment by GabeAl** [comment]
Good point. I've moved all databases except the default to my site last
release for this reason... Maybe it will help at least going forward.
